docs/reference/fprime-translations.md

```diff
@@ -0,0 +1,57 @@
+# F PRime Translation Guide: Software Engineering Terminology to F Prime Nomenclature
+
+This guide provides a mapping between common software engineering concepts and their equivalent implementations in the F´ framework. It serves as a reference for developers new to F´ development.
+
+## Data Structures & Containers
+
+| Software Concept | F Prime Equivalent | Notes |
+|-----------------|-------------------|--------|
+| String | Fw::String | Safe string implementation with size limits |
+| Buffer | Fw::Buffer | Memory buffer with size tracking |
+| Queue | Os::Queue | Thread-safe queue implementation |
```

Comment:
Is `Utils/Types/Queue.hpp` worth mentioning?

Reply:
I'd rather avoid those types until we clean them up a touch.